pnpm install -F cursorless-vscode install-local is broken

I use `pnpm install -F cursorless-vscode install-local` in a script to automate building cursorless with my nix and bash language PRs integrated, and it seems like it is broken now:

```bash
❯ pnpm -F cursorless-vscode install-local

> @cursorless/cursorless-vscode@0.28.0 install-local /home/aa/dev/talon/fidgetingbits-cursorless-build/packages/cursorless-vscode
> bash ./scripts/install-local.sh


> @cursorless/cursorless-vscode@0.28.0 esbuild:prod /home/aa/dev/talon/fidgetingbits-cursorless-build/packages/cursorless-vscode
> pnpm run esbuild:base --minify


> @cursorless/cursorless-vscode@0.28.0 esbuild:base /home/aa/dev/talon/fidgetingbits-cursorless-build/packages/cursorless-vscode
> esbuild ./src/extension.ts --conditions=cursorless:bundler --bundle --outfile=dist/extension.cjs --external:vscode --format=cjs --platform=node "--minify"


  dist/extension.cjs  1005.9kb

⚡ Done in 99ms

> @cursorless/cheatsheet-local@0.1.0 build:prod /home/aa/dev/talon/fidgetingbits-cursorless-build/packages/cheatsheet-local
> pnpm webpack --mode=production --node-env=production


> @cursorless/cheatsheet-local@0.1.0 webpack /home/aa/dev/talon/fidgetingbits-cursorless-build/packages/cheatsheet-local
> pnpm -w compile && env NODE_OPTIONS=--loader=ts-node/esm webpack --config ./src/webpack.config.ts "--mode=production" "--node-env=production"


> cursorless@0.1.0 compile /home/aa/dev/talon/fidgetingbits-cursorless-build
> tsc --build && pnpm -r --if-present compile:esbuild

Scope: 24 of 25 workspace projects
packages/cheatsheet compile:esbuild$ esbuild ./src/index.ts --sourcemap --format=esm --bundle --packages=external --outfile=./out/index.js
│   out/index.js      54.0kb
│   out/index.js.map  90.3kb
│ ⚡ Done in 4ms
└─ Done in 30ms
packages/common compile:esbuild$ esbuild ./src/index.ts --sourcemap --format=esm --bundle --packages=external --outfile=./out/index.js
│   out/index.js      104.8kb
│   out/index.js.map  240.6kb
│ ⚡ Done in 11ms
└─ Done in 38ms
packages/neovim-registry compile:esbuild$ esbuild ./src/index.ts --sourcemap --format=esm --bundle --packages=external --outfile=./out/index.js
│   out/index.js      1.0kb
│   out/index.js.map  2.2kb
│ ⚡ Done in 3ms
└─ Done in 32ms
packages/sentence-parser compile:esbuild$ esbuild ./src/index.ts --sourcemap --format=esm --bundle --packages=external --outfile=./out/index.js
│   out/index.js       7.4kb
│   out/index.js.map  17.7kb
│ ⚡ Done in 2ms
└─ Done in 38ms
packages/node-common compile:esbuild$ esbuild ./src/index.ts --sourcemap --format=esm --bundle --packages=external --outfile=./out/index.js
│   out/index.js      15.6kb
│   out/index.js.map  31.5kb
│ ⚡ Done in 3ms
└─ Done in 21ms
packages/vscode-common compile:esbuild$ esbuild ./src/index.ts --sourcemap --format=esm --bundle --packages=external --outfile=./out/index.js
│   out/index.js      4.9kb
│   out/index.js.map  9.9kb
│ ⚡ Done in 3ms
└─ Done in 24ms
packages/cursorless-engine compile:esbuild$ esbuild ./src/index.ts --sourcemap --format=esm --bundle --packages=external --outfile=./out/index.js
│   out/index.js      514.3kb
│   out/index.js.map    1.1mb
│ ⚡ Done in 25ms
└─ Done in 45ms
packages/cursorless-cheatsheet compile:esbuild$ esbuild ./src/index.ts --sourcemap --format=esm --bundle --packages=external --outfile=./out/index.js
│   out/index.js      1.3kb
│   out/index.js.map  3.3kb
│ ⚡ Done in 4ms
└─ Done in 26ms
packages/neovim-common compile:esbuild$ esbuild ./src/index.ts --sourcemap --format=esm --bundle --packages=external --outfile=./out/index.js
│   out/index.js      31.4kb
│   out/index.js.map  72.8kb
│ ⚡ Done in 5ms
└─ Done in 29ms
.../cursorless-everywhere-talon-core compile:esbuild$ esbuild ./src/index.ts --sourcemap --format=esm --bundle --packages=external --outfile=./out/index.js
│   out/index.js      13.8kb
│   out/index.js.map  32.1kb
│ ⚡ Done in 3ms
└─ Done in 21ms
packages/cursorless-tutorial compile:esbuild$ esbuild ./src/index.ts --sourcemap --format=esm --bundle --packages=external --outfile=./out/index.js
│   out/index.js      22.6kb
│   out/index.js.map  47.5kb
│ ⚡ Done in 5ms
└─ Done in 26ms
packages/test-case-recorder compile:esbuild$ esbuild ./src/index.ts --sourcemap --format=esm --bundle --packages=external --outfile=./out/index.js
│   out/index.js      23.0kb
│   out/index.js.map  42.7kb
│ ⚡ Done in 3ms
└─ Done in 26ms
packages/cursorless-everywhere-talon compile:esbuild$ pnpm run esbuild:prod
│ > @cursorless/cursorless-everywhere-talon@1.0.0 esbuild:prod /home/aa/dev/talon/fidgetingbits-cursorless-build/packages/cursorless-everywhere-talon
│ > pnpm run esbuild:base --minify src/mainProduction.ts
│ > @cursorless/cursorless-everywhere-talon@1.0.0 esbuild:base /home/aa/dev/talon/fidgetingbits-cursorless-build/packages/cursorless-everywhere-talon
│ > bash scripts/esbuild.sh "--minify" "src/mainProduction.ts"
│   out/talon.js      416.3kb
│   out/talon.js.map    2.2mb
│ ⚡ Done in 75ms
└─ Done in 590ms
(node:1375040) ExperimentalWarning: `--experimental-loader` may be removed in the future; instead use `register()`:
--import 'data:text/javascript,import { register } from "node:module"; import { pathToFileURL } from "node:url"; register("ts-node/esm", pathToFileURL("./"));'
(Use `node --trace-warnings ...` to show where the warning was created)
(node:1375040) ExperimentalWarning: `--experimental-loader` may be removed in the future; instead use `register()`:
--import 'data:text/javascript,import { register } from "node:module"; import { pathToFileURL } from "node:url"; register("ts-node/esm", pathToFileURL("./"));'
(Use `node --trace-warnings ...` to show where the warning was created)
assets by path ../out/ 2.45 KiB
  assets by path ../out/app/ 1.02 KiB
    assets by path ../out/app/*.ts 618 bytes 2 assets
    assets by path ../out/app/*.map 426 bytes 2 assets
  assets by path ../out/environments/ 551 bytes
    assets by path ../out/environments/*.map 336 bytes 2 assets
    assets by path ../out/environments/*.ts 215 bytes 2 assets
  assets by path ../out/*.ts 463 bytes
    asset ../out/polyfills.d.ts 257 bytes [compared for emit]
    + 2 assets
  assets by path ../out/*.map 455 bytes
    asset ../out/webpack.config.d.ts.map 181 bytes [compared for emit]
    + 2 assets
asset index.html 232 KiB [compared for emit]
asset main.js 232 KiB [compared for emit] [minimized] (name: main) 1 related asset
orphan modules 381 KiB [orphan] 146 modules
runtime modules 698 bytes 4 modules
cacheable modules 358 KiB
  modules by path ../../node_modules/.pnpm/ 157 KiB
    modules by path ../../node_modules/.pnpm/style-loader@3.3.4_webpack@5.91.0_esbuild@0.20.2_webpac...(truncated) 5.84 KiB 6 modules
    modules by path ../../node_modules/.pnpm/react@18.2.0/node_modules/react/ 7.98 KiB 4 modules
    modules by path ../../node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/ 131 KiB 3 modules
    modules by path ../../node_modules/.pnpm/prop-types@15.8.1/node_modules/prop-types/ 2.6 KiB 3 modules
    modules by path ../../node_modules/.pnpm/scheduler@0.23.0/node_modules/scheduler/ 4.33 KiB 2 modules
    modules by path ../../node_modules/.pnpm/css-loader@6.10.0_webpack@5.91.0_esbuild@0.20.2_webpack...(truncated) 2.31 KiB 2 modules
    ../../node_modules/.pnpm/react-string-replace@1.1.1/node_modules/react-string-re...(truncated) 2.75 KiB [built] [code generated]
    ../../node_modules/.pnpm/@fortawesome+free-solid-svg-icons@6.5.1/node_modules/@f...(truncated) 1.07 KiB [built] [code generated]
  modules by path ./src/ 200 KiB
    ./src/index.tsx + 8 modules 184 KiB [built] [code generated]
    ../../node_modules/.pnpm/css-loader@6.10.0_webpack@5.91.0_esbuild@0.20.2_webpack-cli@5.1.4_/node_modules/css-loader/dist/cjs.js!../../node_modules/.pnpm/postcss-loader@8.1.1_postcss@8.
4.38_typescript@5.5.3_webpack@5.91.0_esbuild@0.20.2_webpack-cli@5.1.4_/node_modules/postcss-loader/dist/cjs.js!./src/styles.css 16.2 KiB [built] [code generated]
webpack 5.91.0 compiled successfully in 2920 ms

> @cursorless/cursorless-vscode@0.28.0 populate-dist /home/aa/dev/talon/fidgetingbits-cursorless-build/packages/cursorless-vscode
> my-ts-node ./src/scripts/populateDist/index.ts "--local-install"

Skipping dist/build-info.json
Copying ../../CHANGELOG.md to dist/CHANGELOG.md
Copying ../../LICENSE to dist/LICENSE
Copying ../../NOTICE.md to dist/NOTICE.md
Copying ../../README.md to dist/README.md
Copying ../cheatsheet-local/dist/index.html to dist/cheatsheet.html
Copying ../../cursorless-snippets to dist/cursorless-snippets
Copying ../../fonts/cursorless-glyph.svg to dist/fonts/cursorless-glyph.svg
Copying ../../fonts/cursorless.woff to dist/fonts/cursorless.woff
Copying ../../images/hats to dist/images/hats
Copying ../../data/fixtures/recorded/tutorial to dist/tutorial
/home/aa/dev/talon/fidgetingbits-cursorless-build/packages/cursorless-vscode/src/scripts/populateDist/populateDist.ts:59
            throw Error(`Missing asset: ${fullSource}`);
                  ^


Error: Missing asset: ../cursorless-vscode-tutorial-webview/out/index.js
    at <anonymous> (/home/aa/dev/talon/fidgetingbits-cursorless-build/packages/cursorless-vscode/src/scripts/populateDist/populateDist.ts:59:19)
    at async Promise.all (index 10)
    at run (/home/aa/dev/talon/fidgetingbits-cursorless-build/packages/cursorless-vscode/src/scripts/populateDist/populateDist.ts:21:3)

Node.js v20.12.2
 ELIFECYCLE  Command failed with exit code 1.
/home/aa/dev/talon/fidgetingbits-cursorless-build/packages/cursorless-vscode:
 ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @cursorless/cursorless-vscode@0.28.0 install-local: `bash ./scripts/install-local.sh`
Exit status 1
```

It's fixable by using `pnpm -F cursorless-vscode-tutorial-webview build` in `./packages/cursorless-vscode/scripts/install-local.sh`, so I'll send a PR.

--- a/packages/cursorless-vscode/scripts/install-local.sh
+++ b/packages/cursorless-vscode/scripts/install-local.sh
@@ -6,6 +6,7 @@ set -euo pipefail
 # 1. Build local cursorless, using special extension id to break update chain
 pnpm esbuild:prod
 pnpm -F cheatsheet-local build:prod
+pnpm -F cursorless-vscode-tutorial-webview build
 pnpm populate-dist --local-install
 
 # 2. Bundle the extension

